guides/advanced_guides/configuration.md

```diff
@@ -296,3 +300,39 @@ The engine ignores missing snapshots and does not throw an error in this case.
 **CLI option**: `--ignore-snapshot-if-db-exists`
 
 The engine skips snapshot importation if a database already exists. No error is thrown in this case.
+
+### Dumps folder
+
+**Environment variable**: `MEILI_DUMPS_FOLDER`
+**CLI option**: `--dumps-folder`
+
+Path of the folder where dumps will be created if the [dump route](/references/dump.md#trigger-a-dump-creation) is called.
+
+**Default value**: `dumps/`
+
+### Import dump
+
+**Environment variable**: `MEILI_IMPORT_DUMP`
+**CLI option**: `--import-dump`
+
+Import a dump from the specified path, must be a `.tar.gz` file.
+
+As the data contained in the dump needs to be indexed, the process will take some time to fully import the dump. Only when the dump has been fully imported, MeiliSearch server will start.
+As the data contained in the dump needs to be indexed,
+the process will take time to fully import the dump depending on the size of your dump.
+
+### Dump batch size
+
+**Environment variable**: `MEILI_DUMP_BATCH_SIZE`
+**CLI option**: `--dump-batch-size`
+
+The batch size used in the importation process. The more you index documents in the same batch the less time it takes but more memory is used.
+
+**Example**
```

Comment:
```
**Example**
Imagine you set `--dump-batch-size 1000` and your dump contains `100 600` documents. Instead of indexing all `100 600` docs in one go, the engine will :

1. Index documents `0` -> `999` (1000 docs)
2. Index documents `1000` -> `1999` (1000 docs)
....
100. Index documents `99 000` -> `100 000` (1000 docs)
101. Index documents `100 000` -> `100 600` (600 docs)

This will avoid being out of memory if you run with little memory access. 
```

Do you prefer this ?

Reply:
It is better, but it still feels unrealistic since 1024 is the default and the difference between 1024 and 1000 would be negligible.
Maybe a good solution is to have two examples: one where you have little memory access and a small database so you set to a small batch size, and one where memory isn't a concern and you have a very large index, so you set a large batch size to speed up indexing.

Reply:
I suggest we add that example in the guide. I will make a PR soon enough where we will be able to add more indepth examples